components/RecipeViewer/Ingredients.tsx

```diff
@@ -0,0 +1,34 @@
+import { Flex, Text, Title } from '@mantine/core';
+import { RecipeIngredient } from '@/types/recipes';
+
+type IngredientsProps = {
+  recipeIngredients: RecipeIngredient[];
+};
+
+const Ingredients = ({ recipeIngredients }: IngredientsProps) => {
+  if (!recipeIngredients.length) {
+    return (
+      <Flex direction="column" gap="16">
+        <Title size="h2">Ingredients</Title>
```

Comment:
There's no reason to render this heading multiple times... render in the parent.